src/sentry/incidents/logic.py

```diff
@@ -52,11 +62,58 @@ def create_incident(
             IncidentGroup.objects.bulk_create([
                 IncidentGroup(incident=incident, group=group) for group in groups
             ])
+
+        if status == IncidentStatus.CREATED:
+            activity_status = IncidentActivityType.CREATED
+        else:
+            activity_status = IncidentActivityType.DETECTED
+
+        create_incident_activity(incident, activity_status, include_event_stats_snapshot=True)
     return incident
 
 
-def build_incident_query_params(incident):
-    params = {'start': incident.date_started, 'end': incident.current_end_date}
+def create_incident_activity(
+    incident,
+    activity_type,
+    value=None,
+    previous_value=None,
+    comment=None,
+    include_event_stats_snapshot=False,
+):
+    snapshot = None
+    if include_event_stats_snapshot:
+        initial_period_length = min(
+            timezone.now() - incident.date_started,
+            MAX_INITIAL_INCIDENT_PERIOD,
+        )
+        end = incident.date_started + initial_period_length
+        start = end - (initial_period_length * 8)
+        snapshot = create_event_stat_snapshot(incident, start, end)
+
+    return IncidentActivity.objects.create(
```

Comment:
Does this activity need to end up in the organization stream at some point?

Reply:
I actually have no idea what the organization stream is, could you point me in the right direction?

Reply:
https://sentry.io/organizations/sentry/activity/

Reply:
Added a ticket for this, https://getsentry.atlassian.net/browse/SEN-620. It's low priority compared to other things at the moment.